examples/http-tinygo-outbound-http/tinygo-hello/go.mod

```
@@ -1,9 +1,9 @@
 module github.com/fermyon/spin/templates/spin-http-tinygo-outbound-http
 
-go 1.17
+go 1.20
 
-require github.com/fermyon/spin/sdk/go v0.0.0
+require github.com/fermyon/spin/sdk/go/v2 v2.0.0
 
 require github.com/julienschmidt/httprouter v1.3.0 // indirect
 
-replace github.com/fermyon/spin/sdk/go v0.0.0 => ../../../sdk/go/
+replace github.com/fermyon/spin/sdk/go/v2 v2.0.0 => ../../sdk/go/
```

```suggestion
replace github.com/fermyon/spin/sdk/go/v2 v2.0.0 => ../../../sdk/go/
```
This is a nested component so needs the extra `../`